src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderMultiplexAlgorithm.cpp

```diff
@@ -942,7 +942,8 @@ namespace OpenMS
       throw OpenMS::Exception::FileEmpty(__FILE__, __LINE__, __FUNCTION__, "Error: No MS1 spectra in input file.");
     }
 
-    //TODO allow skipping?
+    // clear chromatograms (otherwise they are used to calculate optimal RT and m/z ranges)
+    exp.getChromatograms().clear();
```

Comment:
k tracked down why we probably excluded TICS from range calculation.
FFMultiPlex fails if the m/z range starts at 0 (e.g. for TICs)
As a workaround I just remove chromatograms.

Reply:
fair solution